general_documentation/dunning_kruger_effect.md

```diff
@@ -0,0 +1,41 @@
+
+
+### The Cause
+The Dunning-Kruger Effect can be difficult to recognize, especially while it is occurring. For example, it can be difficult for a freshman in college, a junior developer, or a new-hire to be consciously aware of mistakes or shortcomings while performing an activity. All things being equal, this observation can be directly related to a lack of experience, skill, or training and not at all related to capability or willingness to learn. 
+
+### Underestimation by Seniors 
+Has someone declared a task as "easy" because it was easy for them? Such an individual may underestimate the difficulty others may have performing a task. Alternatively, an experienced individual can underestimate the difficult he/she will face when performing a task. In short, the person was overly confident in their own abilities unaware of their own lack of knowledge or skill. These underestimations happen because the person has become unaware of his own skills relative to world around him. 
+
+### Underestimation by Juniors
+Have you ever thought a task would be easy only for it to become an insurmountable one? A new-hire may underestimate the difficulty of a similar task because of the "ease" by which she completed a previously similar task. It happens ALL THE TIME! It is very true that "You do not know what you do not know". The new-hire may have been unaware of the challenges that lay ahead, challenges that a more senior hire would have seen. The person might even be a below-average worker who just "got lucky" on a few tasks. Unfortunately, the person became overconfident -- and yes this is the Dunning Kruger effect at its finest!  
+
+### Estimation by Everyone
+Charles Darwin, in his book "The Descent of Man" [^darwin], wrote “Ignorance more frequently begets confidence than does knowledge”. In short, unfamiliarity in a topic can cause a person to be confidence in misinformation.
+
+****I'm a professional race car driver!****  
+On a small scale, individuals can make erroneous decisions based on an estimation of their own abilities, which could be based on a fallacy. For example, a 1986 comparative study on US drivers[^drivers] concluded that 80% of people considered themselves as "above average drivers". This is basically saying that 80% of US drivers considered themselves better than remaining 50% of the population.  That is also like slicing a pizza into a hundred (100) slices and saying eighty (80) of the slices are larger than the remaining fifty (50). I am sorry, come again? Is this why so many US drivers attempt to weave in-and-out of traffic as if they received nascar level training?  
```

Comment:
I feel maybe the last part of this isn't as necessary, the example is great though.

Reply:
Which part can be removed he entire "on a small scale..."?